tests/logs/push/test_ami_push/test_aws_publish_failures.txt

```diff
@@ -4,24 +4,29 @@
 [   DEBUG] 1 Products(AWS provider) in rhsm: RHEL_HOURLY(awstest)
 [    INFO] Uploading /tmp/aws_staged/region-1-hourly/AWS_IMAGES/ami-1.raw to region region-1 (type: hourly, ship: True)
 [    INFO] Image name: RHEL-8.5-RHEL-8.5.0_HVM_BETA-20210902-x86_64-5-Hourly2-GP2
+[   DEBUG] {"accounts": ["secret-1"], "arch": "x86_64", "billing_products": ["code-0001"], "container": "redhat-cloudimg-region-1", "description": "Provided by Red Hat, Inc.", "ena_support": true, "image_name": "RHEL-8.5-RHEL-8.5.0_HVM_BETA-20210902-x86_64-5-Hourly2-GP2", "image_path": "/tmp/aws_staged/region-1-hourly/AWS_IMAGES/ami-1.raw", "root_device_name": "/dev/sda1", "snapshot_name": "RHEL-8.5-RHEL-8.5.0_HVM_BETA-20210902-x86_64-5-Hourly2-GP2", "sriov_net_support": "simple", "virt_type": "hvm", "volume_type": "gp2"}
```

Comment:
Hmm, it looks like changing to use "%s" really did change the format.

Previously when you did `LOG.info(var)` the log message contained e.g. `u` string literals which is python-specific.  Now those are gone and everything is single-quoted, so it could be parseable as JSON.

Maybe that means `LOG.info(var)` does equivalent of `repr(var)` while `LOG.info("%s", var)` does equivalent of `str(var)`.  Either way it looks better to me now.

Reply:
I think it's the mock_debug _logger method in tests/push/test_ami_push.py: line 85 changed the format. I mocked log.debug in this to sort the dict elements using OrderedDict and then dump as json to match it with the test logs and have a similar format.
Logs are printed in this same format when not using "%s".